.github/workflows/release-sims.yml

```diff
@@ -1,8 +1,8 @@
 name: Release Sims
 on: 
-  push: 
+  pull_request: 
     branches: 
-      - /^v.*/
+      - 'rc/**'
```

Comment:
This means the merging branch, right? e.g. `rc0/vX.Y.Z` is being merged into `rc1/v.X.Y.Z` or `release/v.X.Y.Z`

Reply:
Can we make sure this works on `release/v*` branches too?

Reply:
there isn't much value in doing this. other than running the sims twice in a row, once when the pull request from `rc** -> release/**` is created then again when `rc**` is merged and into `release/**`. well we do have unlimited minutes so it doesnt hurt

Reply:
We should only be running log sims on RC PRs (i.e. `rc** -> release/**`).

Reply:
Sorry - this is what I meant:

> We should only be running log sims on RC PRs (i.e. rc** -> release/**).

(I don't know how to actually implement this though, @marbar3778 is the GH Actions Wizard here)

Reply:
updated to be correct